patches/0001-network-Add-callback-for-upstream-authorization.patch

```diff
@@ -289,7 +289,7 @@ index 9bc074f393..8d035b5cad 100644
    if (generic_conn_pool_) {
 +    bool accepted = read_callbacks_->iterateUpstreamCallbacks(generic_conn_pool_->host(), getStreamInfo());
 +    if (!accepted) {
-+      getStreamInfo().setResponseFlag(StreamInfo::ResponseFlag::UnauthorizedExternalService);
++      getStreamInfo().setResponseFlag(StreamInfo::CoreResponseFlag::UnauthorizedExternalService);
```

Comment:
This change could be folded in to the earlier commit updating the patches?

Reply:
good catch, thanks.